### PR TITLE
Pro intro guide fixes

### PR DESCRIPTION
**Pull request by gavincabbage**
Fixes to this guide: https://documentation.codeship.com/pro/quickstart/getting-started/

**Part 1:** Solve Redis image name discrepancy
**Part 3:** Add warning about not committing your AES key - @ethangj is `csnote warn` a real thing? I guessed at that based on the `csnote info`.
**Part 4:** Solve container directory discrepancy

### CODE REVIEW

File: _pro/quickstart/getting-started-part-three.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ First, if you didn't do it on the last step, you'll need to download your projec
 
 ![Downloading AES key]({{ site.baseurl }}/images/gettingstarted/aes_key.png)
 
+{% csnote warn %}
+Be sure to your AES key to your **.gitignore** to prevent it from being committed.
```
Comment (joesiewert) [inline _pro/quickstart/getting-started-part-three.md]: `to add your`

Comment (gavincabbage) [reply _pro/quickstart/getting-started-part-three.md]: Good eye, thanks!

**Comment by gavincabbage** [comment]
Re `csnote warn` I found #1261 which indicates it should be `warning`, so I corrected that.

**Comment by gavincabbage** [comment]
@joesiewert @ethangj 

Could one of you merge this if you're comfortable with my changes? Thanks!

**Pull request merged**